### DISCUSSION TITLE
There is no projects in my flyteconsole UI, who can tell me why?

### DISCUSSION THREAD

**Discussion by StreamSun** [flyteorg/flyte#6035] [Q&A]
<img width="954" alt="Screenshot 2024-11-20 143617" src="https:/
<img width="954" alt="Screenshot 2024-11-20 143841" src="https://github.com/user-attachments/assets/e6ea04a2-2ff7-4b5b-897c-dc315fbab9c7">
/github.com/user-attachments/assets/93f260d3-f830-4b0f-a5ff-cd8830061ea1">

**Comment by davidmirror-ops** [comment]
@StreamSun this may happen if you are using TLS certificates in your Flyte deployment. In such a case, CORS options were not enabled leading to this situation. Upgrading to Flyte 1.14 should resolve the issue. Otherwise please let us know

**Reply by fg91** [reply]
Is this still an issue for you @StreamSun ?
